src/test/kotlin/org/rust/ide/template/postfix/IntoIterPostfixTemplateTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.template.postfix
+
+import com.intellij.testFramework.LightProjectDescriptor
+import org.rust.WithStdlibRustProjectDescriptor
+
+
+class IntoIterPostFixTemplateTest : PostfixTemplateTest(IntoIterPostfixTemplate(RsPostfixTemplateProvider())) {
+    override fun getProjectDescriptor(): LightProjectDescriptor = WithStdlibRustProjectDescriptor
```

Comment:
Let's use `ProjectDescriptor` annotation to specify project descriptor
See example [here](https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/lang/core/resolve/RsStdlibResolveTest.kt#L12)